rest_auth_toolkit/views.py

```diff
@@ -51,12 +51,15 @@ def post(self, request):
         user = deserializer.save()
 
         if self.email_confirmation_class is None:
-            raise MissingSetting('email_confirmation_string')
+            raise MissingSetting('email_confirmation_class')
 
         confirmation = self.email_confirmation_class.objects.create(user=user)
         if get_setting('email_confirmation_send_email', True):
             email_field = user.get_email_field_name()
             send_email(request, user, getattr(user, email_field), confirmation)
+        else:
+            user.is_active = True
+            user.save()
```

Comment:
This wouldn’t be right for this class, but see the RestAuthToolkitMinimal app config class that doesn’t do user activation.